jre_emul/Classes/jni.m

```diff
@@ -455,6 +455,11 @@ static jint ThrowNew(JNIEnv *env, jclass clazz, const char *message) {
   return 0;
 }
 
+static jint EnsureLocalCapacity(JNIEnv *env, jint capacity) {
+  // no-op
+  return 0;
+}
+
```

Comment:
Good call, we don't need it: https://stackoverflow.com/questions/7081853/jni-ensurelocalcapacity-why

Reply:
Yes. That is why I left is a no-op for now. Nonetheless, the declaration of the method is needed in jni_util.m:110 in JNU_NewObjectByName